tox.ini

```diff
@@ -82,32 +91,32 @@ setenv =
 commands =
     python setup.py build
 
-[testenv:lint]
-basepython = python3.9
-changedir = scripts
-deps = -rrequirements-dev.txt
+[testenv:py39-lint]
+changedir = {toxinidir}
+deps = -rrequirements-flake8.txt
+platform = linux
```

Comment:
Adding `platform = linux` here means this will only run when the platform is linux. This prevents it from running on windows and macos--we don't need to run these utility environments on those environments.

Reply:
This probably also means that devs who are using Windows and macOS can't run these environments. I'm the only dev at the moment and I'm using Linux this week, so I'm going to ignore this for now, but I'll write it up as an issue.

Reply:
<grumble> I'll fix it now.

Reply:
For things like vendorverify and docs, we want to run them in CI only on Linux, but when not in CI on whatever platform. I can't figure out a way to do that that doesn't involve writing a bash script or something else that's platform-dependent, so I'm going to skip it for now.